components/mpas-ocean/src/shared/mpas_ocn_wetting_drying.F

```diff
@@ -424,6 +392,91 @@ subroutine ocn_wetting_drying_wettingVelocity(layerThickEdgeFlux, layerThickness
 
    end subroutine ocn_wetting_drying_wettingVelocity !}}}
 
+!***********************************************************************
+!
+!  routine ocn_compute_wetting_velocity_factor_cell
+!
+!> \brief   Computes wettingVelocityFactor at edges of iCell
+!> \author  Giacomo Capodaglio
+!> \date    09/06/2023
+!> \details
+!>  This routine computes wettingVelocityFactor at the edges of iCell
+!
+!-----------------------------------------------------------------------
+   subroutine ocn_compute_wetting_velocity_factor_cell(wettingVelocityFactor, layerThicknessTmp, normalVelocity, iCell, k)!{{{
```

Comment:
```suggestion
   subroutine ocn_compute_wetting_velocity_factor(wettingVelocityFactor, layerThicknessTmp, normalVelocity, iCell, k)!{{{
```
I think it's better to leave off `cell`. It could be confusing since `wettingVelocityFactor` is an edge quantity.

Reply:
I changed it to: `ocn_wetting_velocity_factor_on_cell_edges`